### DISCUSSION TITLE
Related repositories

### DISCUSSION THREAD

**Discussion by EmaCon15** [practical-tutorials/project-based-learning#459] [Q&A]
Hello, awesome people! I'm a beginner programmer learning Java... yes, I know. I'm aware that Java has a reputation for being less beginner-friendly, but I'm not here to discuss that. I'm here because I stumbled upon this amazing repository and thought it could be very useful. However, there's one issue: there are very few project ideas for Java. I would be grateful to anyone who can suggest other repositories like this to get additional project ideas to practice. Thanks to everyone! ❤️

**Comment by ghost** [comment]
Hello! Some repositories that would be good for you to look into are:
- [eugenp/tutorials](https://github.com/eugenp/tutorials): A comprehensive repository with tutorials/ beginner projects on various Java topics, including core Java, Spring, and Hibernate
- [MunGell/awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - contains a section of projects strictly for Java
- [KrishGaur1354/Java-Projects-for-Beginners](https://github.com/KrishGaur1354/Java-Projects-for-Beginners) - amazing developer who uploads example projects he has done so other beginner developers can try the projects out themselves
